routes/api.js

```diff
@@ -74,9 +74,13 @@ var statsShow = function(req, res) {
 };
 
 var matches = function(req, res) {
+  var limitCount = req.params.limit;
+  if (limitCount < 1 || limitCount == null) {
+    limitCount = 12; //Default Value if one is not specified
+  }
```

Comment:
We need a maximum on this value.
How about adding this line here:
`limitCount = Math.min(limitCount, 100);`